standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/AbstractThriftHiveMetastore.java

```diff
@@ -0,0 +1,1336 @@
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public List<String> get_dataconnectors() throws MetaException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public void alter_dataconnector(String name, DataConnector connector)
+      throws MetaException, NoSuchObjectException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public Type get_type(String name) throws MetaException, NoSuchObjectException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public boolean create_type(Type type)
+      throws AlreadyExistsException, InvalidObjectException, MetaException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public boolean drop_type(String type) throws MetaException, NoSuchObjectException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public Map<String, Type> get_type_all(String name) throws MetaException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public List<FieldSchema> get_fields(String db_name, String table_name)
+      throws MetaException, UnknownTableException, UnknownDBException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public List<FieldSchema> get_fields_with_environment_context(String db_name, String table_name,
+      EnvironmentContext environment_context)
+      throws MetaException, UnknownTableException, UnknownDBException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public GetFieldsResponse get_fields_req(GetFieldsRequest req)
+      throws MetaException, UnknownTableException, UnknownDBException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public List<FieldSchema> get_schema(String db_name, String table_name)
+      throws MetaException, UnknownTableException, UnknownDBException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public List<FieldSchema> get_schema_with_environment_context(String db_name, String table_name,
+      EnvironmentContext environment_context)
+      throws MetaException, UnknownTableException, UnknownDBException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public GetSchemaResponse get_schema_req(GetSchemaRequest req)
+      throws MetaException, UnknownTableException, UnknownDBException, TException {
+    throw new UnsupportedOperationException("this method is not supported");
+  }
+
+  @Override public void create_table(Table tbl)
```

Comment:
The @Override is inline in all the methods. May be just reformat the file with your IDE so that it is in line with the rest of the code style.

Reply:
Done